src/tests/ftest/util/run_utils.py

```diff
@@ -542,6 +544,11 @@ def stop_processes(log, hosts, pattern, verbose=True, timeout=60, exclude=None,
         force (bool, optional): if set use the KILL signal to immediately stop any running
             processes. Defaults to False which will attempt to kill w/o a signal, then with the ABRT
             signal, and finally with the KILL signal.
+        full_command (bool, optional): if set match the pattern using the full command with
```

Comment:
Same recommendation
```suggestion
        use_full_command (bool, optional): if set match the pattern using the full command with
```

Reply:
`match_full_pattern` could work too. But `full_command` isn't as misleading in this context, compared to the others